index.html

```diff
@@ -1129,6 +1129,37 @@ <h3>
           this risk.
         </p>
       </section>
+      <section>
+        <h3>
+          Fingerprinting and Data Leakage
+        </h3>
+        <h4>
+          Browser fingerprinting
+        </h4>
+        <p class="issue" data-number="219">
+          The Digital Credentials API exposes information about which
```

Comment:
I disagree that this is a fingerprinting vector. The response to this is just if the User Agent knows the string, and not if any wallet supports it. As such, the mitigation has already been taken care of: it's not linkable to any wallet and can't be used to check if a wallet is actually supported... just that the API might forward the request to the possible wallets.